tutorials/evoked/30_eeg_erp.py

```diff
@@ -373,7 +374,204 @@
+# effects of interest :footcite:`Rousselet2012,VanRullen2011`.
+#
+# If using peak measures, it is critical to visually inspect the data to
+# make sure the selected time window actually contains a peak. Note that
+# `~mne.Evoked.get_peak` will always identify a peak amplitude in the time
+# window being searched. However, the peak that is identified may be incorrect.
+# Instead of a peak, we could just measure the rising edge of a peak, for
+# instance, which is not ideal. The following demonstrates why visual
+# inspection is crucial. Below, we use a known bad time window (.09 to .12
+# seconds) to search for a peak on ``eeg59``.
+
+# Get BAD peak measures
+bad_tmin, bad_tmax = .09, .12
+_, bad_lat_roi, bad_amp_roi = \
+    l_vis_roi.get_peak(mode='pos', tmin=bad_tmin, tmax=bad_tmax,
+                       return_amplitude=True)
+
+# Convert latency and amplitude to msec and microvolts
+bad_lat_roi *= 1e3
+bad_amp_roi *= 1e6
+
+# Print output
+print('** PEAK MEASURES FROM A BAD TIME WINDOW **')
+print('Channel: eeg59')
+print(f'Peak Latency: {bad_lat_roi:.3f} msec')
+print(f'Peak Amplitude: {bad_amp_roi:.3f} \u00B5V')
+
+# %%
+# If all we had were the above values, it would be unclear if they are truly
+# identifying peak or just a the rising edge. However, it becomes clear that
+# the .09 to .12 second time window use for the search is missing the peak
+# on ``eeg59``. This is shown in the bottom panel where we see the bad time
+# window (highlighted in orange) misses the peak (the pink star). In contrast,
+# the time window defined initially (.065 to .115 seconds; highlighted in blue)
+# returns the actual peak instead of a value on the rising edge. Visual
+# inspection will always help you to convince yourself the data returned are
+# really peaks.
+
+# Make an empty figure handle and axis
+fig, (ax1, ax2) = plt.subplots(nrows=2, ncols=1)
+
+# Plot the ERP, actual peak, and the good time window searched
+l_vis_roi.plot(axes=ax1, time_unit='ms', show=False,
+               titles='Bad time window missing peak')
+ax1.plot(lat_roi, amp_roi, marker="*", color='C6')
+ax1.axvspan(bad_tmin * 1e3, bad_tmax * 1e3, facecolor='C1',
+            alpha=.3)
+ax1.set_xlim(-50, 150)  # Show zoomed in around peak
+
+# Plot the ERP, actual peak, and the bad time window searched
+l_vis_roi.plot(axes=ax2, time_unit='ms', show=False,
+               titles='Good time window finding peak')
+ax2.plot(lat_roi, amp_roi, marker="*", color='C6')
+ax2.axvspan(tmin * 1e3, tmax * 1e3, facecolor='C0',
+            alpha=.3)
+ax2.set_xlim(-50, 150)  # Show zoomed in around peak
+plt.tight_layout()
+
+# %%
+# Mean Amplitude
+# --------------
+#
+# Another common practice in ERP studies is to define a component (or effect)
+# as the mean amplitude within a specified time window. One advantage of this
+# approach is that it is less sensitive to high frequency noise (compared to
+# peak amplitude measures) because averaging over a time window is, in essence,
+# a filter.
```

Comment:
Saying that averaging over time is "in essence, a filter" does not actually explain *why* averaging is less sensitive to high-frequency noise.  Try not to assume that the reader has a deep understanding of how filters work or what they are useful for...  for example, specify here that it acts like a *low-pass* filter, and then state what it is about low-pass filters that mitigates high-frequency noise (smoothing out fast fluctuations in the signal, which may be high amplitude and thus may create a bunch of "false peaks").

Reply:
I updated this to make it clear when first mentioning low-pass filters for peak measures. I inserted a reference in the mean amplitude section to link back to that discussion (and also linked to the discussion on filtering).